Lib/test/test_capi/test_long.py

```diff
@@ -34,35 +53,378 @@ def test_compact_known(self):
         self.assertEqual(_testcapi.call_long_compact_api(sys.maxsize),
                          (False, -1))
 
+    def test_long_check(self):
+        """Test PyLong_Check()"""
```

Comment:
These docstrings do not add any information; they just repeat the test name with a slightly different text format. They also clutter the test output when running tests in verbose mode. Please do not pollute tests like this.